integration-cli/docker_api_containers_test.go

```diff
@@ -853,6 +853,17 @@ func (s *DockerSuite) TestContainerApiPostCreateNull(c *check.C) {
 	if out != "" {
 		c.Fatalf("expected empty string, got %q", out)
 	}
+
+	out_memory, err_memory := inspectField(container.Id, "HostConfig.Memory")
```

Comment:
Also sorry I didn't mention it before, but please use camel case for variable names.